Makefile.am

```diff
@@ -233,6 +238,13 @@ container-ci:
 container-shell:
 	$(CONTAINER_ENGINE) run -it $(CONTAINER_TEST_ARGS) $(CI_TEST_ARGS) $(CI_NAME):$(CI_TAG)
 
+container-rpm-test:
+	$(CONTAINER_ENGINE) run -it $(CONTAINER_TEST_ARGS) $(CI_TEST_ARGS) -v $(CI_DOCKERFILE)/run-build-and-arg:/run-build-and-arg $(RPM_NAME):$(CI_TAG) sh -exc ' \
+	    /run-build-and-arg make rpms; \
+	    cd /tmp/anaconda; \
+	    make run-rpm-tests-only; \
+	    dnf install -y result/build/01-rpm-build/*.rpm'
```

Comment:
I've tried to add this as Makefile variable the same way as we have for ``container-ci`` target but I failed there because it always lost quotes along the way. However, I must say this works pretty well too.

It's really unfortunate that with bash you are still fighting with quoting :(.